internal/vm/task_test.go

```diff
@@ -328,50 +332,52 @@ func TestTaskManager_CreateExecDeleteTask(t *testing.T) {
 	tm := NewTaskManager(shimCtx, logger.WithField("test", t.Name()))
 	ts := &mockTaskService{}
 
-	mockTask := newMockProc("fakeTask", "", mockIOConnector, mockIOConnector, mockIOConnector)
+	require := require.New(t)
```

Comment:
Out of curiosity. Why do we need this? As far as I see `require.New` just wraps `t`, so why not to call `require.X` directly?

Reply:
Passing testing.T to newMockProc() is another way. I'm fine with both ways, while having a contextual require is slightly shorter though.